bionic/code_hasher.py

```diff
@@ -93,67 +82,75 @@ def _ingest_raw_prefix_and_bytes(self, type_prefix, obj_bytes=b""):
         # The fix for this issue can also be applied to dedup code
         # references.
 
-    def _check_and_ingest(self, obj, code_context=None):
+    def _check_and_hash(self, obj, code_context=None):
         """
-        Checks for circular references before calling the _ingest
+        Checks for circular references before calling the _update_hash
```

Comment:
Can we update the docstrings to reflect what is being returned and passed around? It's hard for me to follow what state these hash objects are supposed to be in before and after each method.